File: Bot/examples/License.php
function checkLicense($license, $product, $version) {
    $url = 'http://<your_ip>:<port>/api/client';
    $apiKey = 'API_KEY';

    $data = array(
        'license' => $license,
        'product' => $product,
        'version' => $version,
    );

    $headers = array(
        'Authorization: ' . $apiKey,
    );

    $ch = curl_init();
    curl_setopt($ch, CURLOPT_URL, $url);
    curl_setopt($ch, CURLOPT_POST, 1);
    curl_setopt($ch, CURLOPT_POSTFIELDS, http_build_query($data));
    curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
    curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);

    $response = curl_exec($ch);
    curl_close($ch);

    $responseData = json_decode($response, true);

    if ($responseData['status_overview'] !== 'success' && $responseData['status_code'] !== 200) {
        return false;
    }

    return $responseData;
}

File: Dashboard/examples/License.c
#include <stdio.h>
#include <curl/curl.h>

int main(void) {
    CURL *curl;
    CURLcode res;

    curl_global_init(CURL_GLOBAL_DEFAULT);

    curl = curl_easy_init();
    if (curl) {
        struct curl_slist *headers = NULL;
        headers = curl_slist_append(headers, "Authorization: API_KEY");

        curl_easy_setopt(curl, CURLOPT_URL, "http://<your_ip>:<port>/api/client");
        curl_easy_setopt(curl, CURLOPT_POST, 1L);
        curl_easy_setopt(curl, CURLOPT_HTTPHEADER, headers);

        const char *data = "license=<LICENSE>&product=<PRODUCT>&version=<VERSION>";
        curl_easy_setopt(curl, CURLOPT_POSTFIELDS, data);

        res = curl_easy_perform(curl);

        if (res != CURLE_OK) {
            fprintf(stderr, "curl_easy_perform() failed: %s\n", curl_easy_strerror(res));
        }

        curl_slist_free_all(headers);
        curl_easy_cleanup(curl);
    }

    curl_global_cleanup();

    return 0;
}